builder/config/load.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+
+	"github.com/uor-framework/client/builder/api/v1alpha1"
+)
+
+func ReadConfig(configName string) (c v1alpha1.DataSetConfiguration, err error) {
```

Comment:
These named result parameters don't seem to be used. Should probably be removed since this allowed for naked returns.